corehq/util/bounced_email_manager.py

```diff
@@ -137,6 +138,9 @@ def _get_aws_info(self, message, uid):
                     destination=mail_info.get('destination', []),
                 ))
         else:
+            datadog_counter(
+                'commcare.util.bounced_email_manager.unknown_notification_type'
```

Comment:
nit: I'd omit `util` here (and below) and go with just `commcare.bounced_email_manager.unknown_notification_type`. I say that assuming the `util` is there to mirror the file structure, and that's not really a convention we have, and wouldn't hold up well under a refactor, etc.

Reply:
done